versions/3.0.md

```diff
@@ -1945,7 +1945,7 @@ An object to hold data types that can be consumed and produced by operations. Th
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.
+<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.  The following characters are legal in the `definitionsName`: `a-zA-Z0-9.-_()[]\`.  All other characters, including high-level characters, are not legal in the `definitionsName`.  The name may be used to namespace the definitions--it is the tooling vendor's job to ensure that the namespace is honored.
```

Comment:
I like both @jasonh-n-austin 's suggestions. Myself, I hesitate about including `()[]\`, since the characters `.-_` give a fair bit of punctuation to work with already.